config/config-example.js

```diff
@@ -210,6 +210,11 @@ exports.appealurl = '';
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// ripgrepmodlog - whether to use ripgrep to handle modlog queries
+// If no value is set, the first modlog query after each restart will check if
+// ripgrep binaries are available and, if so, they will be used
+exports.ripgrepmodlog = undefined;
```

Comment:
I mean I guess it should check so it doesn't crash, but I want to avoid checking every time. I suppose I could split it into two variables, `ripgrepmodlog` and `checkedForBinaries`? The latter would probably not be in `Config` though, not sure where else I'd put it.